flutter/lib/sentry_flutter.dart

```diff
@@ -9,4 +13,97 @@ mixin SentryFlutter {
     final String version = await _channel.invokeMethod('getPlatformVersion');
     return version;
   }
+
+  static void _flutterErrorIntegration(Hub hub, SentryOptions options) {
+    final defaultOnError = FlutterError.onError;
+
+    FlutterError.onError = (FlutterErrorDetails errorDetails) async {
+      options.logger(
+          SentryLevel.debug, 'Capture from onError ${errorDetails.exception}');
+
+      // TODO: create mechanism
+
+      await hub.captureException(
+        errorDetails.exception,
+        stackTrace: errorDetails.stack,
+      );
+
+      // call original handler
+      if (defaultOnError != null) {
+        defaultOnError(errorDetails);
+      }
+
+      // do we need this?
+      // print('Capture from FlutterError ${details.exception}');
+      // Zone.current.handleUncaughtError(details.exception, details.stack);
+    };
+  }
+
+  static void _isolateErrorIntegration(Hub hub, SentryOptions options) {
+    final receivePort = RawReceivePort(
+      (dynamic error) async {
+        options.logger(SentryLevel.debug, 'Capture from IsolateError $error');
+
+        // is it a list? cond. error.length != 2 is probably wrong
+        if (error is List<dynamic> && error.length != 2) {
+          dynamic stackTrace = error[1];
+          if (stackTrace != null) {
+            stackTrace = StackTrace.fromString(stackTrace as String);
+          }
+          await Sentry.captureException(error[0], stackTrace: stackTrace);
+        }
+      },
+    );
+
+    Isolate.current.addErrorListener(receivePort.sendPort);
+  }
+
+  static Integration _runZonedGuardedIntegration(
+      Function callback,
+  ) {
+    void integration(Hub hub, SentryOptions options) {
+      runZonedGuarded(() {
```

Comment:
```suggestion
       runZonedGuarded(() {
```

Reply:
the IDE always rollback this change, its correct already